machine_management/control_plane_machine_management/cpmso-configuration.adoc

```diff
@@ -97,6 +97,13 @@ Some sections of the control plane machine set CR are provider-specific. The fol
 //Sample VMware vSphere provider specification
 include::modules/cpmso-yaml-provider-spec-vsphere.adoc[leveloffset=+2]
 
+//Sample VMware vSphere failure domain configuration
+include::modules/cpmso-yaml-failure-domain-vsphere.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* For an example of an infrastructure resource specification that defines resources for each failure domain, see xref:../../post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc#specifying-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration[Specifying multiple regions and zones for your cluster on vSphere]
```

Comment:
The only place I see `infrastructures.config.openshift.io` with a name, it is called the [cluster-wide infrastructure CRD](https://docs.openshift.com/container-platform/4.13/post_installation_configuration/post-install-vsphere-zones-regions-configuration.html#references-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration). I don't see _infrastructure resource specification_ being used.

Reply:
:heavy_check_mark: